rewriter/DefDelegator.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SORBET_REWRITER_DEF_DELEGATOR_H
+#define SORBET_REWRITER_DEF_DELEGATOR_H
+#include "ast/ast.h"
+
+namespace sorbet::rewriter {
+
+/**
+ * This class implements Forwardable#def_delegator
+ * by desugaring things of the form
+ *
+ *    class MyClass
+ *      def_delegator :thing, :foo
+ *      def_delegator :thing, :bar, :aliased_bar
+ *    end
+ *
+ * into
+ *
+ *    class MyClass
+ *      sig {params(arg0: T.untyped).returns(T.untyped)}
+ *      def foo(*arg0); end
+ *
+ *      sig {params(arg0: T.untyped).returns(T.untyped)}
+ *      def aliased_bar(*arg0); end
```

Comment:
I think these should be:
```suggestion
 *      sig {params(arg0: T.untyped, blk: T.nilable(Proc)).returns(T.untyped)}
 *      def foo(*arg0, &blk); end
 *
 *      sig {params(arg0: T.untyped, blk: T.nilable(Proc)).returns(T.untyped)}
 *      def aliased_bar(*arg0, &blk); end
```

Reply:
You’re right, this was copy/pasted from the description of the “delegate” rewriter which also should have a block arg so I’ll update both.